airbyte-integrations/connectors/source-mysql/src/main/java/io/airbyte/integrations/source/mysql/MySqlCdcConnectorMetadataInjector.java

```diff
@@ -19,6 +22,14 @@ public void addMetaData(final ObjectNode event, final JsonNode source) {
     event.put(CDC_LOG_POS, source.get("pos").asLong());
   }
 
+  public void addMetaDataToRowsFetchedOutsideDebezium(final ObjectNode record, final String transactionTimestamp,
```

Comment:
Which change is this? If it's this PR : I think it's still inflight : https://github.com/airbytehq/airbyte/pull/28756

Reply:
so likely @nguyenaiden will have to make the same change here, since I might merge this before him

Reply:
This [thread](https://airbytehq-team.slack.com/archives/C03C4AVJWG4/p1690319289798629)

Reply:
Same one. Since Duy is OOO today/Monday that PR will need to adde that column here too. Will comment there as well. For now it should be safe as this code is disabled by default